servers/republik/graphql/resolvers/_mutations/discussion/submitComment.js

```diff
@@ -85,6 +94,8 @@ module.exports = async (_, args, { pgdb, user, t, pubsub }) => {
       skipUndefined: true
     })
 
+    await notify(comment, discussion, context)
```

Comment:
sure that this needs to be awaited? How long does it take to notify 100 people?

Reply:
why not after slack? and also not await slack? IMO if slack is down we don't want to fail a comment submit.

Reply:
sending webNotifications and emails is not awaited in `servers/republik/lib/Notifications.js`. But I added it there and removed it here for more clarity.